packages/devtools_app/lib/src/primitives/auto_dispose.dart

```diff
@@ -25,19 +23,19 @@ class Disposer {
 
   /// Track a stream subscription to be automatically cancelled on dispose.
   void autoDisposeStreamSubscription(StreamSubscription subscription) {
-    if (subscription == null) return;
     _subscriptions.add(subscription);
   }
 
   /// Track a focus node that will be automatically disposed on dispose.
-  void autoDisposeFocusNode(FocusNode node) {
+  void autoDisposeFocusNode(FocusNode? node) {
     if (node == null) return;
     _focusNodes.add(node);
   }
 
   /// Add a listener to a Listenable object that is automatically removed when
   /// cancel is called.
-  void addAutoDisposeListener(Listenable listenable, [VoidCallback listener]) {
+  void addAutoDisposeListener(Listenable? listenable,
+      [VoidCallback? listener]) {
```

Comment:
nit: add a trailing comma after the last parameter. General rule of thumb: if a method signature or method call has to break to a second line, always use the trailing comma. For Flutter code, we always use a trailing comma (with the exception of a few widgets like SizedBox or a Text widget to name a couple).

here and elsewhere in this PR